pkg/config/config_template.yaml

```
@@ -733,6 +733,11 @@ api_key:
   #
   # compression_level: 6
 
+  ## @param batch_wait - integer - optional - default: 5
+  ## The maximum time the Datadog Agent waits to fill each batch (either in content size or number of logs).
```

I think this wording may imply that the setting can be set to the `content size or number of logs`. What about:
```suggestion
  ## The maximum time the Datadog Agent waits to fill each batch of logs before sending.
```